SETUP.md

```diff
@@ -1,44 +1,80 @@
-# Create an app
-- First, we recommend [creating a new workspace](https://slack.com/get-started#create) where you won’t disrupt real work getting done.
-- [Create a Slack app](https://api.slack.com/apps/new) and install it to the new workspace you created.
-- Hit the ```Create App``` button and you’ll land on your app’s **Basic Information** page.
-![](https://slack.dev/bolt-js/assets/basic-information-page.png)
+*   First, we recommend [creating a new workspace](https://slack.com/get-started#create) where you won’t disrupt real work getting done.
+
+*   [Create a Slack app](https://api.slack.com/apps/new) and install it to the new workspace you created.
+
+*   Click on  `Create App` button to go to the **Basic Information** page.
+    ![](https://slack.dev/bolt-js/assets/basic-information-page.png)
 
 # Tokens and installing app
-- Navigate to the **OAuth & Permissions** on the left sidebar and scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**. CHAOSS Slack bot currently has these [OAuth Scopes](https://docs.google.com/document/d/1NJd-nNKUNb3Q0lRb5cfmUU8kpRcYGjh-vPqpk4CCvic/edit#heading=h.v1ah7sirde2a) enabled, so enable those for full functionality of the bot. If you're adding a new function, you might need more permissions. A comprehensive list of permission scopes is available at [api.slack.com/scopes](https://api.slack.com/scopes).
-- Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You’ll be led through Slack’s OAuth UI, where you should allow your app to be installed to your development workspace.
-- Once you authorize the installation, you’ll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
-![](https://slack.dev/bolt-js/assets/bot-token.png)
+
+*   Navigate to the **OAuth & Permissions** on the left sidebar.
+    ![image](https://github.com/peculiaruc/peculiaruc.github.io/assets/35475543/07be85b8-7b28-4691-aab6-7d847b35c18e)
+
+*   Scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**. CHAOSS Slack bot currently has these \[OAuth Scopes]
```

Comment:
This whole section "CHAOSS Slack bot currently has these [OAuth Scopes](https://docs.google.com/document/d/1NJd-nNKUNb3Q0lRb5cfmUU8kpRcYGjh-vPqpk4CCvic/edit#heading=h.v1ah7sirde2a) enabled, so enable those for full functionality of the bot. If you're adding a new function, you might need more permissions. See the set up for adding new functions below." should be together, below the images.

Reply:
Resolved